src/styles/reviews.scss

```diff
@@ -0,0 +1,43 @@
+@import 'variables';
+
+.reviews {
+  display: flex;
+  box-sizing: border-box;
+  margin: 0 auto 24px;
+  width: 166px;
+  height: 16px;
+  text-align: center;
+  justify-content: space-between;
+  padding: 2px 0 0 2px;
+
+  &__text {
+    font-family: $font-roboto;
+    font-weight: $font-medium;
+    font-size: 10px;
+    line-height: 14px;
+    color: $color-font;
+    width: 53px;
+    height: 16px;
+    text-align: right;
```

Comment:
Similarly, 'width' and 'height' should come before 'text-align' in the '.reviews__text' class.